tsconfig.build.json

```diff
@@ -15,5 +15,6 @@
     "isolatedModules": true,
     "noEmit": true
   },
-  "include": ["src"]
+  "include": ["src/**/*.ts", "src/**/*.tsx"],
+  "exclude": ["src/**/*.js", "src/**/*.jsx"]
```

Comment:
also exclude all `node_modules` folders, all `build` folders, and all `dist` folders

Reply:
Done